src/Cdn/Cdn/help/Disable-AzCdnCustomDomainHttps.md

```diff
@@ -0,0 +1,217 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Cdn.dll-Help.xml
+Module Name: Az.Cdn
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cdn/disable-azcdncustomdomainhttps
+schema: 2.0.0
+---
+
+# Disable-AzCdnCustomDomainHttps
+
+## SYNOPSIS
+Disables Custom Domain HTTPS.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Disable-AzCdnCustomDomainHttps -CustomDomainName <String> -EndpointName <String> -ProfileName <String>
+ -ResourceGroupName <String> [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByObjectParameterSet
+```
+Disable-AzCdnCustomDomainHttps -InputObject <PSCustomDomain> [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceIdParameterSet
+```
+Disable-AzCdnCustomDomainHttps -ResourceId <String> [-PassThru] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Disable-AzCdnCustomDomainHttps -CustomDomainName $customDomainName -EndpointName $endpointName -ProfileName $profileName -ResourceGroupName $resourceGroupName
+true
```

Comment:
true is only returned when PassThru is specified, so you can remove it from all examples.

Reply:
removed